template/build/dev-server.js

```diff
@@ -1,3 +1,5 @@
+if (!process.env.NODE_ENV) process.env.NODE_ENV = 'development'
```

Comment:
Good point. I still think it'd be a good idea to set to `config.dev.env` though, instead of directly to `'development'`.